glass/shells.py

```diff
@@ -45,28 +45,28 @@
 
 from __future__ import annotations
 
-import collections.abc
 import itertools
-import typing
 import warnings
+from collections.abc import Callable, Sequence
+from typing import TYPE_CHECKING, NamedTuple
 
 import numpy as np
-import numpy.typing as npt
+from numpy.typing import NDArray
 
 from glass.core.algorithm import nnls
 from glass.core.array import ndinterp
 
-if typing.TYPE_CHECKING:
+if TYPE_CHECKING:
     from cosmology import Cosmology
 
-ArrayLike1D = collections.abc.Sequence[float] | npt.NDArray[np.float64]
-WeightFunc = typing.Callable[[ArrayLike1D], npt.NDArray[np.float64]]
+ArrayLike1D = Sequence[float] | NDArray[np.float64]
+WeightFunc = Callable[[ArrayLike1D], NDArray[np.float64]]
```

Comment:
Can these go in the TYPE_CHECKING block?

Reply:
In my experience, only imports really go inside `TYPE_CHECKING`

Reply:
The `TYPE_CHECKING` block is for everything that doesn't matter at runtime. An example I saw earlier is e.g. [here](https://github.com/python/mypy/blob/8104d0198813a59dd26c8e9f50779533c60292b4/mypyc/build.py#L50-L59) in mypy itself. There are far fewer type aliases than type imports, so maybe that's the cause?

Reply:
If it causes no harm then happy to do, will raise a PR now

Reply:
@paddyroddy Causes no harm either way, so maybe we just do it if we ever touch that code again?

Reply:
https://github.com/glass-dev/glass/pull/520